Doc/library/sqlite3.rst

```diff
@@ -1153,6 +1153,10 @@ Blob Objects
 
       .. literalinclude:: ../includes/sqlite3/blob.py
 
+   A :class:`Blob` can also be used as a :term:`context manager`:
```

Comment:
Should we put this example first? I feel like it should be more idiomatic.

Reply:
Sounds good to me.

Reply:
Hm, those two examples are very similar. I wonder if it would be better to consolidate them.

Also, I think we should move the blob class section up and put them just after the row class section; I think it makes sense to group the class sections.

Reply:
> Hm, those two examples are very similar. I wonder if it would be better to consolidate them.

I reworded the text and consolidated the examples in ca093f3aae984581edc1fe1c68879f658dc1d039. I think the example could be improved still.

Reply:
> Also, I think we should move the blob class section up and put them just after the row class section; I think it makes sense to group the class sections.

If you are ok with the recent change, I can reorder the blob section in a new commit. Or we can do another PR for that.

Reply:
Agree with reordering and consolidating to a single example.

I would suggest making the example show only the `with` version. I feel like that should be the main interface we direct people to, and the example doesn't have to show every possible operation you can do with a blob. Similarly, if I was writing an introduction to files in Python, I'd just show `with open() as f:` and not mention the explicit `f.close()` call prominently.

Reply:
Agreed. And I'll also move the example up to the description.

Reply:
Done: bce9c9fbda257021f464a2ce5e214cf40d66e223 I think we're getting close to something.